.gitignore

```diff
@@ -31,7 +31,6 @@ npm-debug.log
 tmp
 .map
 Thumbs.db
-*.*-e
```

Comment:
we want to keep the line in the gitignore, so you can't commit it into git even if it mistakenly makes it out of release.sh